frontend/pages/projects/[id].tsx

```diff
@@ -46,14 +52,20 @@ const ProjectDetails = (): JSX.Element => {
                 <label>
                     <span>Start</span>
                     <input type="text" defaultValue={currentProject.startDate?.toLocaleDateString()} />
+                    <CalendarSelector onSelectedDateChange={setCurrentStartDate}/>
                 </label>
                 <label>
                     <span>End</span>
                     <input type="text" defaultValue={currentProject.endDate?.toLocaleDateString()} />
+                    {currentStartDate >= currentEndDate ? <p> it doesnt make sense </p> : <p></p>}
```

Comment:
it doesnt make sense ist nicht gut. Dir faellt bestimmt ein besserer Text ein, optimal mit Hinweis, was der Benutzer besser machen kann, z.B. the end has to be after the start